### PR TITLE
Remove unnecessary e2e db and hasura instances from actions

### PR DESCRIPTION
**Pull request by ghost**
- e2e tests are not run in parallel anymore, so there is no need for extra instaces of these services
- Use run-cypress-tests-v1 in this repo also, so tests are run in one thread

- - -
This change is [Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-tools/22)

### CODE REVIEW

**Comment by ghost** [comment]
Create branches for `setup-e2e-environment-v2` and `seed-infrastructure-links-v2` after this has been merged. `seed-infrastructure-links-v1` has been only a git tag.

File: .github/workflows/test-run-cypress-tests-action.yml [hunk 1]
```diff
@@ -21,8 +21,6 @@ jobs:
         uses: ./github-actions/seed-infrastructure-links
 
       - name: Run e2e tests from github action
-        uses: ./github-actions/run-cypress-tests
+        uses: HSLdevcom/jore4-tools/github-actions/run-cypress-tests@run-cypress-tests-v1
```
Comment (culka) [inline .github/workflows/test-run-cypress-tests-action.yml:25]: Referencing an old version of the script may be quite confusing, there should be a comment why this is done.


On the other hand, the v2 of the `run-cypress-tests` action seems to have a few other differences in addition to parallel execution. `--verbose` flag and some `grepTags` at least. I'm not sure if they worked a bit differently with `cypress-parallel`, but maybe the action should also be updated.

**Review by culka — CHANGES_REQUESTED** [review]

**Review by ghost — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-tools/22)* status: 0 of 3 files reviewed, 1 unresolved discussion (waiting on @culka)

___
*[`.github/workflows/test-run-cypress-tests-action.yml` line 24 at r1](https://reviewable.io/reviews/HSLdevcom/jore4-tools/22#gh-1406408858:-NkJU01e5ETwTgwV_wHB:b-p7vphl) ([raw file](https://github.com/HSLdevcom/jore4-tools/blob/3c6ef336af7c57e8a6fedef490737fcf4a8360d9/.github/workflows/test-run-cypress-tests-action.yml#L24)):*
<details><summary><i>Previously, culka (Teemu Mäkinen) wrote…</i></summary><blockquote>

Referencing an old version of the script may be quite confusing, there should be a comment why this is done.


On the other hand, the v2 of the `run-cypress-tests` action seems to have a few other differences in addition to parallel execution. `--verbose` flag and some `grepTags` at least. I'm not sure if they worked a bit differently with `cypress-parallel`, but maybe the action should also be updated.
</blockquote></details>

Reverting to an old version of the action was needed since we wanted to move back to running tests in one thread. This is already done in all other repositories, I think. Should we add comments to all other repositories as well? The --verbose flag is related only to the parallel execution. grepTags works only with v1.

A new version of the action could be the same as v1, but we could include the 'video' parameter from v2. But I think that is a separate issue.

**Review by ghost — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-tools/22)* status: 0 of 3 files reviewed, 1 unresolved discussion (waiting on @culka)

___
*[`.github/workflows/test-run-cypress-tests-action.yml` line 24 at r1](https://reviewable.io/reviews/HSLdevcom/jore4-tools/22#gh-1406408858:-NkJh-l8CK2SMjxbtLdL:b-aqtmbc) ([raw file](https://github.com/HSLdevcom/jore4-tools/blob/3c6ef336af7c57e8a6fedef490737fcf4a8360d9/.github/workflows/test-run-cypress-tests-action.yml#L24)):*
<details><summary><i>Previously, sr-vala (Samuli Rasi) wrote…</i></summary><blockquote>

Reverting to an old version of the action was needed since we wanted to move back to running tests in one thread. This is already done in all other repositories, I think. Should we add comments to all other repositories as well? The --verbose flag is related only to the parallel execution. grepTags works only with v1.

A new version of the action could be the same as v1, but we could include the 'video' parameter from v2. But I think that is a separate issue.
</blockquote></details>

Added a comment about referencing the remote version of this repository in the action.

**Review by culka — APPROVED** [review]
:lgtm: 

> Reviewed 2 of 3 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-tools/22#-:-NkJiFU_9VaHVAyl1DiJ:bg7igkv)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @sr-vala)

___
*[`.github/workflows/test-run-cypress-tests-action.yml` line 24 at r1](https://reviewable.io/reviews/HSLdevcom/jore4-tools/22#gh-1406408858:-NkJi9vq3NxnEmuO4WVt:b-9755fn) ([raw file](https://github.com/HSLdevcom/jore4-tools/blob/3c6ef336af7c57e8a6fedef490737fcf4a8360d9/.github/workflows/test-run-cypress-tests-action.yml#L24)):*
<details><summary><i>Previously, sr-vala (Samuli Rasi) wrote…</i></summary><blockquote>

Added a comment about referencing the remote version of this repository in the action.
</blockquote></details>

Thank you!

**Review by jarkkoka — APPROVED** [review]
:lgtm: 

> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-tools/22#-:-NkZVLwl5VJFVTZfNG3U:bg7igkv)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @sr-vala)

**Pull request merged**